corehq/form_processor/abstract_models.py

```diff
@@ -37,6 +37,10 @@ def set_submission_properties(self, submission_post):
     def to_json(self):
         raise NotImplementedError()
 
+    @property
+    def is_normal(self):
```

Comment:
yeah i think we can use `PositiveSmallIntegerField` for the `is_*`. we should be cautious though, i used to work at a place where one dude made lots of properties based on one field and it gets super complex pretty quickly. especially when you're looking at raw sql